### DISCUSSION TITLE
Grateful for an Inspiring Meeting with Alper..

### DISCUSSION THREAD

**Discussion by IVEN2024** [ant-media/Ant-Media-Server#6824] [General]
This week in Istanbul, my son and I had the pleasure of meeting Alper. It was an exceptional meeting filled with valuable ideas and insights. Alper was incredibly receptive to suggestions and feedback, showcasing his openness and collaboration. The AMS team has truly impressed me with their outstanding support and innovation. I sincerely hope we get the chance to meet again in the future. It was a highly informative and inspiring experience. Thank you, Alper and the entire AMS team, for your fantastic product and unwavering support.

Best regards,
Iven

**Comment by yashtandon113** [comment]
Hi @IVEN2024 
I hope you are doing well.

Thank you for your kind words and valuable feedback! We are delighted to hear about your positive experience with **Alper** and the AMS team. It is always a delight to engage with our users and share thoughts. We look forward to supporting you further and hope to meet again soon!

Thanks to you, @akiziltoprak 👍
